client/client.go

```diff
@@ -293,3 +295,38 @@ func (lc *LogClient) postAndParse(path string, body []byte, rsp interface{}) (*H
 	}
 	return httpData, nil
 }
+
+// AddChain performs an add-chain request, posting the provided certificate
+// chain to the CT Log hosted at LogClient.url.  The first certificate in chain
+// should be the end-entity certificate, with the second chaining to the first
+// and so on to the last, which should either be the root certificate or a
+// certificate that chains to a root certificate that is accepted by the Log.
+// Returned is:
+//   - a populated ct.SignedCertificateTimestamp, if no error is returned.
+//   - an HTTPData struct (see above).
```

Comment:
The docs describing that an HTTPData struct is returned even when err!=nil are quite a long way above.
You might add a little more here, e.g.
`//   - an HTTPData struct (even when err != nil, see above).`